package.json

```diff
@@ -29,7 +29,7 @@
 		"clean-examples": "node .github/scripts/cleanup-examples-changesets.mjs",
 		"update-examples": "npx tsx .github/scripts/update-examples.ts",
 		"ci:version": "changeset version && node .github/scripts/cleanup-examples-changesets.mjs && pnpm install --no-frozen-lockfile",
-		"snapshot": "node .github/scripts/release-snapshot.js"
+		"snapshot": "changeset --empty && changeset version --snapshot && changeset publish --no-git-tag --snapshot"
```

Comment:
I think this is the old script. Should revert these changes.

Reply:
@msaaddev what do you think? @ahmadbilaldev tag who you're asking the question from?

Reply:
@ahmadbilaldev you are correct. These are old changes. Need to revert this. The following is the latest script:

```
"snapshot": "node .github/scripts/release-snapshot.js"
```

Reply:
I have fixed it.